server/routes/inPrisonCoursesAndQualifications/index.ts

```diff
@@ -18,6 +18,7 @@ export default (router: Router, services: Services) => {
     router.get('/plan/:prisonNumber/in-prison-courses-and-qualifications', [
       checkUserHasViewAuthority(),
       retrievePrisonerSummaryIfNotInSession(services.prisonerSearchService),
+      retrieveCuriousInPrisonCourses(services.curiousService),
```

Comment:
Retrieve the Curious In Prison Courses as part of the request handler chain, rather than in the controller itself 👍